test/parallel/test-fs-close-fast-api.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+
+// A large number is selected to ensure that the fast path is called.
```

Comment:
Is it ensured that 100_000 is enough?
Any chance to really verify that fast API is actually used?

Reply:
You can try something similar to https://chromium.googlesource.com/v8/v8/+/e3e8ea5d657b886201d4e1982589a72f77909ba4/test/mjsunit/compiler/fast-api-helpers.js#7 though it needs to be paired with specific V8 optimizer flags and can be subject to changes as V8 changes their optimizing strategy.

Reply:
mjsunit.js exists in the node tree, you should be able to load it into this test with `--allow-natives-syntax` and then you can use stuff like `assertOptimized` and it will continue to work as v8 is updated.